### DISCUSSION TITLE
功能看着挺好，但是真忍不住吐槽，文档像x一样···找个配置还的自己看源码

### DISCUSSION THREAD

**Discussion by machachong** [antvis/G2#5140] [Q&A]
##### Checklist

- [ ] `npm test` passes
- [ ] benchmarks are included
- [ ] commit message follows commit guidelines
- [ ] documents are updated

**Comment by hustcc** [comment]
6 月有有 1~2 周优化文档，主要是人力上不够，开源不易。如果有兴趣，可以一起来提建议和想法，一起优化！
